wp-includes/sqlite-ast/class-wp-sqlite-driver.php

```diff
@@ -1149,6 +1172,8 @@ private function translate( $ast ) {
 					return null;
 				}
 				return $this->translate_sequence( $ast->get_children() );
+			case 'defaultCollation':
+				return null;
```

Comment:
Can we actually support anything else than UTF-8? If not, we could make this an error for non-utf8 values. Or a warning at least.

Reply:
@adamziel Good point! I just removed it here because SQLite doesn't support `COLLATE = ...` on tables, and I'm storing the exact value in the information schema. I think we'll need to allow all the `utf8_something`, `utf8mb3_something`, and `utf8mb4_something` collations at least, although I suppose we won't be able to sort the values the same way as MySQL does. We could also consider allowing single-byte charsets, etc.

Reply:
Makes sense, thank you! Let's make it an error for now, then, and document what you just said inline to make sure we give this the attention it needs in the future.